packages/kit/src/runtime/load.js

```diff
@@ -5,9 +5,20 @@
 export function normalize(loaded) {
 	// TODO should this behaviour be dev-only?
 
-	if (loaded.error) {
-		const error = typeof loaded.error === 'string' ? new Error(loaded.error) : loaded.error;
+	if (
+		loaded.error ||
+		(loaded.status && loaded.status >= 400 && loaded.status <= 599 && !loaded.redirect)
+	) {
 		const status = loaded.status;
+		if (!loaded.error) {
+			console.warn(' "status" property was returned from load() without an error ');
```

Comment:
I don't think this should be a warning. it seems totally valid to me. I'd either remove it or make it `debug` (same below on line 33). If you do keep it, you should also removing the leading/trailing spaces